src/main/kotlin/com/wafflestudio/toyproject/team4/core/user/api/UserController.kt

```diff
@@ -36,6 +37,17 @@ class UserController(
         @UserContext username: String,
     ) = ResponseEntity(userService.getPurchases(username), HttpStatus.OK)
 
+    @Authenticated
+    @PostMapping("/me/purchases")
+    fun postPurchases(
+        @RequestBody request: PurchasesRequest,
+        @RequestHeader(value = "Authorization") authorization: String,
+        @UserContext username: String,
+    ) {
+        userService.postPurchases(username, request)
+        return
+    }
```

Comment:
다음과 같이 수정하면 불필요한 `return`을 생략할 수 있을 것 같습니다.
```suggestion
fun postPurchases(...) = userService.postPurchases(...)
```